tool/deploy.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+
+import 'package:grinder/grinder.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+main(List args) => grind(args);
+
+@DefaultTask('Deploy software')
+deploy() {
```

Comment:
That actually seemed to raise an exception, so I just made `publish.dart` (formerly `deploy.dart`) part of `atom.grind` instead.